src/Plugin/KeyInput/ApigeeAuthKeyInput.php

```diff
@@ -119,15 +123,33 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
     if (empty($values['organization'])) {
       $form['password']['#states']['required'] = [$state_for_public, $state_for_private];
     }
+
+    $state_for_not_gcp_hosted = [];
+
+    if (GceServiceAccountAuthentication::isAvailable()) {
+      $form['gcp_hosted'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Use the default service account if this portal is hosted on GCP'),
+        '#description' => $this->t("Please ensure you have added 'Apigee Developer Administrator' role to the default compute engine service account hosting this portal."),
```

Comment:
I'm not familiar enough with this, but is "Apigee Developer Administrator" the only way the role can be named?

Reply:
This is what they named it. 

Documented here => https://www.drupal.org/docs/contributed-modules/apigee-edge/configure-the-connection-to-apigee-edge#s-set-the-service-account-roles